plugin/trino-teradata/src/test/java/io/trino/plugin/integration/TestTeradataJDBCTypeMapping.java

```diff
@@ -0,0 +1,225 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.integration;
+
+import io.trino.plugin.integration.clearscape.ClearScapeEnvironmentUtils;
+import io.trino.testing.datatype.SqlDataTypeTest;
+import org.junit.jupiter.api.Test;
+
+import java.sql.SQLException;
+
+import static io.trino.spi.type.BigintType.BIGINT;
+import static io.trino.spi.type.CharType.createCharType;
+import static io.trino.spi.type.DateType.DATE;
+import static io.trino.spi.type.DecimalType.createDecimalType;
+import static io.trino.spi.type.DoubleType.DOUBLE;
+import static io.trino.spi.type.IntegerType.INTEGER;
+import static io.trino.spi.type.SmallintType.SMALLINT;
+import static io.trino.spi.type.TimeType.createTimeType;
+import static io.trino.spi.type.TimeWithTimeZoneType.createTimeWithTimeZoneType;
+import static io.trino.spi.type.TimestampType.createTimestampType;
+import static io.trino.spi.type.TinyintType.TINYINT;
+import static io.trino.spi.type.VarbinaryType.VARBINARY;
+import static io.trino.spi.type.VarcharType.createVarcharType;
+import static io.trino.type.JsonType.JSON;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+public class TestTeradataJDBCTypeMapping
```

Comment:
Rename to TestTeradataTypeMapping.

Also, please follow our test guideline. https://trino.io/docs/current/develop/tests.html#conventions-and-recommendations
* Test classes should be defined as package-private and final.
* Test methods should be defined as package-private.

Reply:
addressed in new PR https://github.com/trinodb/trino/pull/26731